src/scribe_data/cli/get.py

```diff
@@ -136,7 +142,14 @@ def get_data(
     # MARK: Emojis
 
     elif data_type in {"emoji-keywords", "emoji_keywords"}:
-        generate_emoji(language=language, output_dir=output_dir)
+        if not check_if_pyicu_installed():
```

Comment:
By adding this user don't have to execute the command again. 
![image](https://github.com/user-attachments/assets/cf26d035-fd18-4db2-af95-e5600aa66bb5)

But I don't know why I got `403 error` and `EOFError`